src/_includes/layouts/projects.njk

```diff
@@ -6,7 +6,24 @@
     <ul id="content" class="content grid-layout posts-{{ eleventyNavigation.key | lower }}">
         {# Featured Projects #}
         {% for project in posts %}
-           {% include "partials/components/projects-item.njk" %}
+            {#
+                Project items are included directly here rather than in a partial
+                template due to an apparent race condition with the Sharp plugin.
+            #}
+            <li class="post card-layout corner-effect">
+                {% if project.data.thumbnailFilename %}
+                    {% if project.data.thumbnailAltText %}
+                        {% set thumbnailAltText = project.data.thumbnailAltText %}
+                    {% else %}
+                        {% set thumbnailAltText = "" %}
```

Comment:
When an alt text is missing, an accessibility error will occur. There are 2 solutions off the top of my head but I'm not sure if they are good solutions:

1. Set the thumbnail image to a decorative image;
2. Blur the image which serves as a warning that the alt text for this image is missing. This was the solution taken on the WeCount website - https://github.com/inclusive-design/wecount.inclusivedesign.ca/blob/dev/src/scss/layout/_layout.scss#L203-L208

Reply:
Setting `alt=""` should be the equivalent of making an image presentational. However not including the `alt` attribute at all would be an a11y failure.

https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
This is an interesting consideration. My thinking behind the way I programmed it was that if the author chose not to supply an alt text, then the image is considered decorative in which case an empty alt attribute is an acceptable choice according to [WCAG technique H67](https://www.w3.org/TR/WCAG20-TECHS/H67.html):

> If no title attribute is used, and the alt text is set to null (i.e. `alt=""`) it indicates to assistive technology that the image can be safely ignored.

I wonder if it's worth adding a check for when `thumbnailAltText` is present but empty, though this could also be due to an active choice by an author to supply an empty alt text attribute (i.e. deliberately following H67)...

Reply:
Great to know. Thanks, guys.